resources/js/views/Dashboard.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div class="dashboard container-fluid">
+    <div v-if="this.userGroup === 'control'">
```

Comment:
instead of checking the userGroup state variable, check the group inside the user object, same applies for the other instances where this is checked